source/agora/common/Config.d

```diff
@@ -174,6 +175,9 @@ public struct NodeConfig
     /// The duration between requests for retrieving the latest blocks
     /// from all other nodes
     public Duration block_catchup_interval = 20.seconds;
+
+    /// The amount of a penalty for slashed validators
+    public Amount slash_penalty_amount = Amount(100_000_000_000);
```

Comment:
How about adding a new commit which implements a `coins` function, which returns:

```D
Amount coins (in ulong num_coins)
{
    Amount(UnitPerCoin.value * num_coins, true);
}
```

And then it can be used as:

```D
public Amount slash_penalty_amount = 10_000.coins;
```

MinFreezeAmount does something similar, but it uses the private `value` field. So I suggest adding a helper function.

Reply:
Also it would be easier if the slashing value is in the config is in coins rather than units.

Reply:
> How about adding a new commit which implements a coins function which returns:
> Amount coins (in ulong num_coins)

Okay, it would be good. I will add it.